packages/dtd-catalog-exporter/src/index.ts

```diff
@@ -0,0 +1,16 @@
+import {
+  initFileManager,
+  logger,
+  ReadModelRepository,
+} from "pagopa-interop-commons";
+import { config } from "./config/config.js";
+import { readModelServiceBuilder } from "./services/readModelService.js";
+import { dtdCatalogExporterServiceBuilder } from "./services/dtdCatalogExporterService.js";
+
+await dtdCatalogExporterServiceBuilder({
+  readModelService: readModelServiceBuilder(ReadModelRepository.init(config)),
+  fileManager: initFileManager(config),
+  loggerInstance: logger({
+    serviceName: "dtd-catalog-exporter",
+  }),
```

Comment:
It could be worth also generating and passing a correlationId so that it's logged in all logs of the same export invocation
```suggestion
  loggerInstance: logger({
    serviceName: "dtd-catalog-exporter",
    correlationId: crypto.randomUUID(),
  }),
```